### DISCUSSION TITLE
delete

### DISCUSSION THREAD

**Discussion by SethWiiPlaza** [community/community#45561] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Can the GitHub staff please work harder to support all of the iOS jailbreak developers/hackers and possibly add a detailed error reporting system for them, so there is no need for the shitty outdated tweak '_Cr4shed_'

I want to see more support for these backwoods underground jailbreak devs from the GitHub organization itself. All of the active iOS jailbreak developers that crank out these remarkable hacks/mods/tweaks/exploits etc., for iOS deserve a more immersive sharing system & community than the garbage r/jailbreak community 
![howto-install-sileo-cydia-zebra-installer-odyssey-1](https://user-images.githubusercontent.com/8001327/215157847-219ea27c-f755-4107-aed3-83b9d6fefa16.jpg)

**Comment by cehfisher** [comment]
Hi @SethWiiPlaza - It looks like your issue was mistakenly added to the accessibility discussion board. @queenofcorgis could you please move this to a different category? Thanks!

**Comment by SethWiiPlaza** [comment]
//
